functional/FunctionalTest.java

```diff
@@ -2755,130 +2738,90 @@ public static void composeObject_test2() throws Exception {
       listSourceObjects.add(s2);
-
-      client.composeObject(bucketName, destinationObjectName, listSourceObjects, null, null);
-      Files.delete(Paths.get(filename1));
-      Files.delete(Paths.get(filename2));
-
-      client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(filename1).build());
-      client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(filename2).build());
-      client.removeObject(
-          RemoveObjectArgs.builder().bucket(bucketName).object(destinationObjectName).build());
-
-      mintSuccessLog(
-          "composeObject(String bucketName, String objectName,List<ComposeSource> composeSources, "
-              + "Map <String,String > headerMap, ServerSideEncryption sseTarget)",
-          "with offset and length.",
-          startTime);
-
-    } catch (Exception e) {
-      ErrorResponse errorResponse = null;
-      if (e instanceof ErrorResponseException) {
-        ErrorResponseException exp = (ErrorResponseException) e;
-        errorResponse = exp.errorResponse();
-      }
-
-      // Ignore NotImplemented error
-      if (errorResponse != null && errorResponse.errorCode() == ErrorCode.NOT_IMPLEMENTED) {
-        mintIgnoredLog(
-            "composeObject(String bucketName, String objectName,List<ComposeSource> composeSources, "
-                + "Map <String,String > headerMap, ServerSideEncryption sseTarget)"
-                + "with offset and length.",
-            null,
-            startTime);
-      } else {
-        mintFailedLog(
-            "composeObject(String bucketName, String objectName,List<ComposeSource> composeSources, "
-                + "Map <String,String > headerMap, ServerSideEncryption sseTarget)",
-            "with offset and length.",
-            startTime,
-            null,
-            e.toString() + " >>> " + Arrays.toString(e.getStackTrace()));
-        throw e;
+      try {
+        client.composeObject(
+            ComposeObjectArgs.builder()
+                .bucket(bucketName)
+                .object(destinationObjectName)
+                .sources(listSourceObjects)
+                .build());
+        client.removeObject(
+            RemoveObjectArgs.builder().bucket(bucketName).object(destinationObjectName).build());
+      } finally {
+        deleteFilesAndObjects(bucketName, new String[] {filename1, filename2});
       }
+      mintSuccessLog(methodName, mintArgs, startTime);
+    } catch (Exception e) {
+      handleException(methodName, mintArgs, startTime, e);
     }
   }
 
-  /**
-   * Test: composeObject(String bucketName, String objectName, List&lt;ComposeSource&gt;
-   * composeSources,Map &lt;String, String&gt; headerMap, ServerSideEncryption sseTarget).
-   */
+  /** Test: composeObject(ComposeObjectArgs args) with one source. */
   public static void composeObject_test3() throws Exception {
+    String methodName = "composeObject(ComposeObjectArgs args)";
+    String testTags = "with one source";
```

Comment:
Did you mean `testArgs` ?

Reply:
These are just tags not the arguments for mint so `testArgs` seems logical. Its done in `copyObject` too.